### PR TITLE
fix handlebars and stylus indentation

### PR DESCRIPTION
**Pull request by ghost**
corrects .hbs and .styl files to have 2 spaces of indentation, along with the rest of the site

### CODE REVIEW

**Comment by hiroppy** [comment]
LGTM :+1:

**Comment by phillipj** [comment]
@tentakel could we enforce correct indentation going forward? Fixing all of this isn't all that valuable if changes could be merged w/wrong indentation, and I don't think we should spend time doing indentation nitpicking in PRs manually.

**Comment by ghost** [comment]
@phillipj sure, i'd more than welcome that. should i add that to the `CONTRIBUTING.md` or make a new file for that?

**Comment by fhemberger** [comment]
Well, personally, I'm not a big fan of 2 space indentation. Things use to get sloppy over time, which can lead to closing tags missing in HTML and so on.

But if everybody is okay with it, I'm fine.

@tentakel You rearranged some code in a few files. What's the reason for that? Would be easier to review the PR if it was just the indentation in one commit and the other changes in a separate one.

Please also change the `.editorconfig` to reflect the new indentation: https://github.com/nodejs/nodejs.org/blob/master/.editorconfig#L7

@phillipj `.editorconfig` is one place, not sure if we should/can add a linter checking all files for indentation.

**Comment by ghost** [comment]
@fhemberger i did? 😨 all i remember is fixing one `<div>` that was supposed to be a closing tag. sorry, but could you point out an example?

**Comment by phillipj** [comment]
@tentakel was thinking more in terms of _enforcing_ as part of the `npm test` script as we do with standard for .js files. Maybe give [htmllint-cli](https://www.npmjs.com/package/htmllint-cli) a shot?

**Comment by fhemberger** [comment]
@tentakel Uh, never mind. The GitHub diff got a little confused because of all the indentation change.

**Comment by ghost** [comment]
@phillipj that sounds like a good idea. i'll look into how to do that

**Comment by ghost** [comment]
alright, htmllint is running together with standard. i've made sure to keep consistency with the standard patterns, too

File: package.json [hunk 1]
```diff
@@ -63,6 +63,7 @@
   },
   "devDependencies": {
     "faucet": "^0.0.1",
+    "htmllint-cli": "0.0.4",
```
Comment (phillipj) [inline package.json]: Use `^` as the rest of our devDeps?

Comment (ghost) [inline package.json]: will do!

**Comment by phillipj** [comment]
@tentakel nice! Care to try https://www.npmjs.com/package/stylint#indentpref--default-false-number-or-false- while you're at it to get .styl indentation enforced as well?

**Comment by ghost** [comment]
@phillipj yep!

**Comment by ghost** [comment]
stylint is added and runs smoothly

File: .htmllintrc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (stevemao) [inline .htmllintrc:1]: I wish there were similar projects as `standard` so that we don't need to create two extra metadata files.

**Comment by phillipj** [comment]
LGTM 👍

**Pull request merged**